apis/python/src/tiledbsoma/__init__.py

```diff
@@ -133,16 +133,17 @@
     except OSError:
         # Otherwise try loading by name only.
         ctypes.CDLL(libtiledbsoma_name)
-
 from somacore import (
-    Axis,
-    CoordinateSpace,
     AffineTransform,
-    ScaleTransform,
-    IdentityTransform,
-    UniformScaleTransform,
     AxisColumnNames,
     AxisQuery,
+    CoordinateSpace,
```

Comment:
shouldn't these remain in sorted order?  (lint)

Reply:
Weird -- pre-commit ran cleanly ... 👀

Reply:
that is worth investigating as well (at some point) ...

Reply:
💯

Reply:
Good catch, https://github.com/single-cell-data/TileDB-SOMA/pull/3495 should fix this.